HelpMyStreetFE/HelpMyStreetFE/Repositories/CommunityRepository.cs

```diff
@@ -798,50 +799,50 @@ private CommunityViewModel GetBostonGNS(string pLanguage)
                 },
                 new FlagViewModel()
                 {
-                    Name = "Romana",
+                    Name = "Română",
                     ImageLocation = "/img/community/bostongns/flags/romania-flag-small.png",
                     Language = "Romanian"
                 },
                 new FlagViewModel()
                 {
-                    Name = "Bulgaria",
+                    Name = "български",
                     ImageLocation = "/img/community/bostongns/flags/bulgaria-flag-small.png",
                     Language = "Bulgarian"
                 },
                 new FlagViewModel()
                 {
-                    Name = "Portuges",
+                    Name = "Português",
                     ImageLocation = "/img/community/bostongns/flags/portugal-flag-small.png",
                     Language = "Portugese"
                 }
             };
 
-
+            communityViewModel.Flags.First(f => f.Language.Equals(language)).IsSelected = true;
 
             communityViewModel.CommunityVolunteers = new List<CommunityVolunteer>()
             {
                 new CommunityVolunteer()
                 {
-                    Name = "Proudly funded by The National Lottery Community Fund",
+                    Name = "",
```

Comment:
Should Name and Role be null rather than ""?  Looks like the markup checks for null explicitly.